docs/guides/continuous-integration/github-actions.mdx

```diff
@@ -111,8 +111,8 @@ jobs:
 
 To try out the example above yourself, fork the
```

Comment:
My only hesitation to do this is that there are already workflows in the Kitchen Sink and `single.yml` that already has `name: Cypress tests`. So adding this new workflow will likely cause a lot of noise in the GHA tab on github. It has almost the same name and all the other ones will be triggered with a new push as we discuss in the section below this. 

Let's sync up and see if we have a better repo to recommend. Otherwise, we may want to add a note about having this as the only workflow file so it's less confusing.

Reply:
Sounds good. Additionally, based on [this GitHub Issue comment](https://github.com/cypress-io/github-action/issues/634#issuecomment-1437410549) from @MikeMcC399, I think it will also be better if we reverted the node versions to reflect the workaround for [`wait-on` with node18+](https://github.com/cypress-io/github-action#wait-on-with-nodejs-18) and will reflect the version used in the [Cypress RWA `main.yml`](https://github.com/cypress-io/cypress-realworld-app/blob/develop/.github/workflows/main.yml).

Reply:
@conversayShawn 

I don't think that Node.js `16` should be used as an example. It is no longer the LTS version and it transitions to [end-of-life](https://github.com/nodejs/release#release-schedule) on Sep 11, 2023.

The [cypress-io/cypress-example-kitchensink](https://github.com/cypress-io/cypress-example-kitchensink) CI workflows have all been updated to use Node.js `18`.

The `wait-on` issue should not stop anybody using Node.js `18`. Unfortunately [Cypress RWA](https://github.com/cypress-io/cypress-realworld-app) is quite out of date. It is using `react-scripts` from [create-react-app](https://github.com/facebook/create-react-app) which is not being maintained (last update was one year ago) and installing RWA with `yarn` gives pages full of deprecation warnings. It should probably be migrated to use `Next.js` instead of `react-scripts`. This would also solve the `wait-on` issue. It should also be the goal to have [Cypress RWA](https://github.com/cypress-io/cypress-realworld-app) running on the LTS version of Node.js (currently `18`, moving to `20` later this year).

Reply:
Makes sense. I have updated to node 18 and resolved the conflicts listed below.